### PR TITLE
Port Tomorrow Theme

### PR DESCRIPTION
**Pull request by n8tlarsen**
First of all, nvim-highlite is a real gem among nvim theme packages. The ability to port existing themes or easily make new ones is a treat. I've used [Tomorrow Theme](https://github.com/chriskempson/tomorrow-theme) for a number of years now, but it lacks Treesitter support so I'm adding it here.

### CODE REVIEW

File: lua/highlite/color/palette/tomorrow-night-blue.lua [hunk 1]
```diff
@@ -0,0 +1,256 @@
...
+	bg2    = 0x003F8E
+	gray   = 0x7285B7
+	red    = 0xFF9DA4
+	red_hi = 0xF72F31
+	orange = 0xFFC58F
+	yellow = 0xFFEEAD
+	green  = 0xD1F1A9
+	aqua   = 0x99FFFF
+	blue   = 0xBBDAFF
+	purple = 0xEBBBFF
```
Comment (Iron-E) [inline lua/highlite/color/palette/tomorrow-night-blue.lua]: We should put `local` before these variables as well, or else they will be set globally :)

File: lua/highlite/color/palette/tomorrow-night-bright.lua [hunk 2]
```diff
@@ -0,0 +1,256 @@
...
+	bg2    = 0x424242
+	gray   = 0x969896
+	red    = 0xD54E53
+	red_hi = 0xF72f31
+	orange = 0xE78C45
+	yellow = 0xE7C547
+	green  = 0xB9CA4A
+	aqua   = 0x70C0B1
+	blue   = 0x7AA6DA
+	purple = 0xC397D8
```
Comment (Iron-E) [inline lua/highlite/color/palette/tomorrow-night-bright.lua]: See comment on `tomorrow-night-blue` palette

File: lua/highlite/color/palette/tomorrow-night-eighties.lua [hunk 3]
```diff
@@ -0,0 +1,256 @@
...
+	bg2    = 0x515151
+	gray   = 0x999999
+	red    = 0xEB7375
+	red_hi = 0xF72f31
+	orange = 0xF99157
+	yellow = 0xFFCC66
+	green  = 0x99CC99
+	aqua   = 0x66CCCC
+	blue   = 0x6699CC
+	purple = 0xCC99CC
```
Comment (Iron-E) [inline lua/highlite/color/palette/tomorrow-night-eighties.lua]: See comment on `tomorrow-night-blue` palette

File: lua/highlite/color/palette/tomorrow-night.lua [hunk 4]
```diff
@@ -0,0 +1,256 @@
...
+	bg2    = 0x373b41
+	gray   = 0x969896
+	red    = 0xCC6666
+	red_hi = 0xF72F31
+	orange = 0xDE935F
+	yellow = 0xF0C674
+	green  = 0xB5BD68
+	aqua   = 0x8ABEB7
+	blue   = 0x81A2BE
+	purple = 0xB294BB
```
Comment (Iron-E) [inline lua/highlite/color/palette/tomorrow-night.lua]: See comment on `tomorrow-night-blue` palette

File: lua/highlite/color/palette/tomorrow.lua [hunk 5]
```diff
@@ -0,0 +1,256 @@
...
+	bg2    = 0xD6D6D6
+	gray   = 0x8E908C
+	red    = 0xC82829
+	red_hi = 0xF72F31
+	orange = 0xF5871F
+	yellow = 0xEAB700
+	green  = 0x718C00
+	aqua   = 0x3E999F
+	blue   = 0x4271AE
+	purple = 0x8959A8
```
Comment (Iron-E) [inline lua/highlite/color/palette/tomorrow.lua]: See comment on `tomorrow-night-blue` palette

**Review by Iron-E — CHANGES_REQUESTED** [review]
Thank you for the kind words :)

I hoped that this plugin would be able to serve this sort of purpose. If you found the documentation for this process unclear in any way, I would be happy to receive feedback!

I only have a small nit, and then I'll be happy to merge whenever you are happy with it.

**Comment by n8tlarsen** [comment]
@Iron-E I'm not super familiar with Lua so thanks for the suggestion. Should be ready to merge now.

**Review by Iron-E — APPROVED** [review]

**Comment by Iron-E** [comment]
Glad I could help 🙂 thanks for the PR!

**Pull request merged**